### ISSUE TITLE
Allow default text in StylableTextBox

### ISSUE THREAD

**Issue by Nockiro** [Assorted-Development/winforms-stylable-controls#8]
Currently, it's not possible to predefine a text that is not a `Hint` on the StylableTextBox control.
Since this is kind of a self-made issue (due to the `Hint` being an extension to the original control) and the original control allows setting a default value, `StylableTextBox` should imho also allow this.

**Comment by wolframhaussig** [comment]
Good catch, this should definitely be fixed! This would remove the default Text as soon as the user focuses this textbox as the control thinks that this is the hint.
